crates/context/src/handlers/execute.rs

```diff
@@ -286,11 +286,61 @@ async fn internal_execute(
         });
     };
 
+    let precompiled_bytes = node_client
+        .get_precompiled_application_bytes(&context.application_id)
+        .await?;
```

Comment:
PS: prefer directly calling `get_blob_bytes` so we don't have to do a db lookup on every execution

see `ContextManager::get_or_fetch_context`

and you can make `ContextMeta` this:

<img src="https://github.com/user-attachments/assets/aecafe1a-31bf-4623-8023-6c562fedf8ab" width="50%" />